netiquette/README.md

```diff
@@ -0,0 +1,63 @@
+# BcnEng's slack netiquette
+
+Welcome to BcnEng's slack netiquette. If you don't agree with some guideline or whish to propose an improvement, feel free to open a pull request.
+
+## Definition
+
+From [wikipedia](https://en.wikipedia.org/wiki/Etiquette_in_technology#Netiquette):
+
+> Netiquette [...] is a set of social conventions that facilitate interaction over networks [...]
+
+## Conventions
+
+Here you will find a set of basic conventions that we find useful.
+
+### behaviour
+
+Be respectful with your colleagues and follow the [Code of Conduct](https://github.com/bcneng/bcneng/tree/master/coc) rules.
+
+### accounts
+
+As a people-first slack organization, company accounts are not allowed.
+
+### content
+
+Use threads when possible to keep the conversations organized.
+
+Pasting a random link sometimes does not add much value. Consider explaining why you did it.
+
+Consider removing the link previews.
+
+Use emoticons to give a quick feedback. Writing _haha_, _jaja_, _XDDD_ or _lol_ don't contribute much.
+
+The administrators reserve the rights to:
+- remove abusive comments, but always warning the publisher before
+- remove link previews, for readibility's sake
+
+### channels
+
+Prefer a specific channel over #general or #random, everyone is auto-joined in those channels and we are a lot of people.
+
+Ask in #general or #offtopic-staff before creating a new channel, maybe it was already created and archived because it didn't have traffic. Some channels were merged, but they can be un-merged if needed. Some channels will be archived if there is no traffic in them.
+
+Read the channel topic for some guidelines in specific channels. Here you are some examples:
```

Comment:
TODO: Add the channel dedicated to the coc and netiquette questions. https://github.com/bcneng/bcneng/pull/10/files#r176035196